src/control/server/mgmt_svc_test.go

```diff
@@ -303,12 +217,17 @@ func TestPoolGetACL_Success(t *testing.T) {
 		t.Errorf("Expected no error, got: %v", err)
 	}
 
-	if resp == nil {
-		t.Fatal("Expected a response, got nil")
+	// Avoid comparing the internal Protobuf fields
+	isHiddenPBField := func(path cmp.Path) bool {
+		if strings.HasPrefix(path.Last().String(), ".XXX_") {
+			return true
+		}
+		return false
+	}
+	cmpOpts := []cmp.Option{
+		cmp.FilterPath(isHiddenPBField, cmp.Ignore()),
```

Comment:
This is kinda cool, hadn't seen it before. At some point (not for this PR), we may want to take a look through the tests to see if we can extract some universal options into a common.CmpOpts() or something like that.